string name
string display_name
bool end_interaction
bool is_fallback